src/main/java/domain/Race.java

```diff
@@ -0,0 +1,80 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class Race {
+    final private List<Car> cars;
+    private List<Car> winners;
+
+    //domain.Race 생성자 : ","를 이용해 구분하여 레이스에 참가하는 차량의 이름을 입력
+    public Race(String nameOfCars) throws IllegalArgumentException {
+        String[] carNames = nameOfCars.split(",");
+
+        cars = new ArrayList<>();
+        for (String carName : carNames) {
+            cars.add(new Car(carName));
+        }
+    }
+
+    //getter
+    public List<Car> getWinner() {
+        return winners;
+    }
+
+    //레이스 실행
+    public void runRace(int round){
+        //주어진 횟수동안 이동
+        for(int i=0; i<round; i++){
+            moveAllCars();
+        }
+
+        //승자 구하기
+        checkWinners(calcMaxLocation());
+    }
+
+    //모든 차량 이동
+    private void moveAllCars(){
+        for(Car car: cars){
+            car.move(new Random().nextInt(10));
+            printCarProgress(car);
+        }
+        System.out.println();
+    }
+
+    private void printCarProgress(Car car){
+        System.out.print(car.getName() + " : ");
+        for(int i = 0; i < car.getLocation(); i++){
+            System.out.print("-");
+        }
+        System.out.println();
+    }
+
+    //최대 거리 구하기
+    private int calcMaxLocation(){
+        int maxLocation = -1;
+        for(Car car:cars){
+            maxLocation = Math.max(maxLocation, car.getLocation());
+        }
+
+        return maxLocation;
+    }
+
+    //승자 구하기
+    private void checkWinners(int maxLocation){
+        winners = new ArrayList<>();
+
+        for(Car car: cars){
+            addWinners(car, maxLocation);
+        }
+    }
+
+    //승자 추가하기
+    private void addWinners(Car car, int maxLocation){
+        if(car.getLocation() == maxLocation){
+            winners.add(car);
+        }
+    }
```

Comment:
`checkWinners`이라는 메소드명과 다르게 void를 반환하고 있습니다 !
아래 작성된 addWinners의 메소드의 if 조건이 check에 더 가까운 느낌도 있습니다. 

addWinners 메소드를 checkWinners로 변경 후 boolean을 반환하는 방식으로 변경하고, checkWinners 메소드를 addWinners로 변경 후 리스트에 넣는 방식을 사용한다면 메소드 명과 반환형을 일치시킬 수 있을 거 같다는 생각이 듭니다 !

```suggestion
    private void addWinners(Car car, int maxLocation){
        if (checkWinners(car, maxLocation)) {
            winners.add(car);
        }
    }

    private boolean checkWinners(Car car, int maxLocation){
        return car.getLocation() == maxLocation;
    }
```

Reply:
메서드 명명이 제일 어려운 것 같아요 😢 
메소드명과 필요인자만 보고서도 메서드가 하는 일과 반환형을 추론할 수 있도록 작성해보겠습니다!